lib/src/parser/models/flavorizr.dart

```diff
@@ -50,7 +50,7 @@ class Flavorizr {
   final String assetsUrl;
 
   @JsonKey()
-  final IDE? ide;
+  final List<IDE>? ide;
```

Comment:
This way it'll be a breaking change. it might be good to support the old `ide` and a new `ides` property and start to deprecate the old one

Reply:
Done :)